src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1

```diff
@@ -228,6 +228,18 @@ class ServicesSecurityStatus: ADOSVTCommandBase
 					$result += $currentResourceResults;
 				}
 				
+				if([Organization]::InstalledextensionInfo)
+				{
+					$years = 2
+					if([Helpers]::CheckMember($svtObject.ControlSettings, "Organization.ExtensionsLastUpdatedInYears"))
+                    {
+                        $years = $svtObject.ControlSettings.Organization.ExtensionsLastUpdatedInYears
+					}
+					$folderpath=([WriteFolderPath]::GetInstance().FolderPath) + "\$($_.ResourceName)"+"_ExtensionInfo.csv";
+					[Organization]::InstalledextensionInfo | Select-Object extensionName,publisherId,KnownPublisher,publisherName,version,@{Name = "Too Old(>$($years)year(s))"; Expression = { $_.Updated } },lastPublished,@{Name = "Sensitive Permissions"; Expression = { $_.scopes} },@{Name = "NonProd (Galleryflags)"; Expression = { $_.ProductionReady}},@{Name = "NonProd (ExtensionName)"; Expression = { $_.Preview }},TopPublisher,PrivateVisibility | Export-Csv -Path $folderpath -NoTypeInformation
```

Comment:
Add comments

Reply:
Added comment for fetching property from controlsettings